From c9157c5072aeeae23e2de25080deeea959ac7dc5 Mon Sep 17 00:00:00 2001
From: Jake Hunsaker <jhunsake@redhat.com>
Date: Wed, 17 Oct 2018 11:45:48 -0400
Subject: [PATCH] [sos_collector] Configure initial sosreport command after
 case id prompt

We were previously configuring the initial sosreport command before the
case id prompt when a user didn't specify either --case-id or --batch on
the commandline, thus only the sos-collector archive would have the case
id and not the sosreports run on the nodes. No other options were
affected by this, and case ids are now properly set on all node
sosreports.

Signed-off-by: Jake Hunsaker <jhunsake@redhat.com>
---
 soscollector/sos_collector.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/soscollector/sos_collector.py b/soscollector/sos_collector.py
index 1e66de2..c0044ec 100644
--- a/soscollector/sos_collector.py
+++ b/soscollector/sos_collector.py
@@ -264,7 +264,6 @@ this utility or remote systems that it connects to.
         self.console.info("\nsos-collector (version %s)\n" % __version__)
         intro_msg = self._fmt_msg(disclaimer % self.config['tmp_dir'])
         self.console.info(intro_msg)
-        self.configure_sos_cmd()
         prompt = "\nPress ENTER to continue, or CTRL-C to quit\n"
         if not self.config['batch']:
             input(prompt)
@@ -332,6 +331,7 @@ this utility or remote systems that it connects to.
             self.config['cluster'].modify_sos_cmd()
         self.get_nodes()
         self.intro()
+        self.configure_sos_cmd()
 
     def intro(self):
         '''Prints initial messages and collects user and case if not
-- 
2.14.4

